plugins/modules/cloud/misc/terraform.py

```diff
@@ -319,12 +319,24 @@ def remove_workspace(bin_path, project_path, workspace):
     _workspace_cmd(bin_path, project_path, 'delete', workspace)
 
 
-def build_plan(command, project_path, variables_args, state_file, targets, state, plan_path=None):
+def build_plan(command, project_path, variables_args, state_file, targets, state, APPLY_ARGS, plan_path=None):
     if plan_path is None:
         f, plan_path = tempfile.mkstemp(suffix='.tfplan')
 
+    local_command = command.copy()
+
     plan_command = [command[0], 'plan', '-input=false', '-no-color', '-detailed-exitcode', '-out', plan_path]
```

Comment:
How about writing this as
```suggestion
    plan_command = [command[0], 'plan']
    plan_command.extend(['-input=false', '-no-color', '-detailed-exitcode', '-out', plan_path)
```
and moving the second line after the two `if`s. Then you don't have to do `insert(2, c)`, but can simply `append(c)`.

Reply:
I think this is a more clean solution for sure, I've added this in to my latest commit.
Thank you!